### ISSUE TITLE
Dead?

### ISSUE THREAD

**Issue by jjwilliams42** [Semantic-Org/Semantic-UI-Angular#61]
So is this repo dead? No activity for a couple of months. Did you hit a roadblock? Need some assistance?

**Comment by DaSchTour** [comment]
I guess there is no big need for this or they are simply waiting for Angular2. Nobody wants to put a lot of work into this when in some month everything has to be rewritten.

**Comment by dv336699** [comment]
wondering the same, although there is still some years for angular1 to be called deprecated.

**Comment by vazh** [comment]
yeah, angular1 is still far from being deprecated.
and angular2 is still far from stable too.

**Comment by hmaesta** [comment]
Someone is already working for Angular 2. https://github.com/vladotesanovic/ngSemantic

**Comment by wesllycode** [comment]
DO NOT GO produce nothing angular About 1 ?

**Comment by ruprechtcua** [comment]
seems dead

**Comment by matheuspoleza** [comment]
Hey, guys. Thank you for worrying about it

I have some points to do:
- **1)** Semantic-UI-Angular is dead?: Yes, for a while
- **2)** We decided to move on angular2 version, but I think this was a not good option
- **3)**  I'm thinking to take another way and do just it to angular1.x
- **4)** If someone want to help, please contact me on gitter.

**Comment by vazh** [comment]
somebody is already did angular2 version, but currently no update since last month.
either way, angular1 is going to be transformed into angular2 step by step.
the route component is the example.
dont rush it, just do it little by little.

unfortunately i cant help at all, writing is the only help i can do. lol.

good luck, cheers.

**Comment by artem-galas** [comment]
It is a pity that he is dead :( just wanted to be a contribution :)

**Comment by matheuspoleza** [comment]
@vazh I don't think that angular1 is going to be transformed to angular2. I think that is there 2 totally different versions and doesn't have so much compatibility. 

In practice, there is a thousand of company and people using ng1 yet and it will take a while to all people move to ng2. 

A possible way to renew this project is take care about ng1, released and after that open a new discussion about ng2

**Comment by vazh** [comment]
@matheuspoleza not really meant it that way, sorry for the confusion. and yeah it will take a while before people moving to ng2, that's why its best to do it in angular1 first, then you can progress into angular2 bit by bit. just do it natively, without jquery.

the thing is, angular1 life cycle is still far from dead. when lots of components released to support angular2 then its time to think about changing to angular2. but right now, i dont see any stable components can be used to build a whole application for people like me who can only use available components, except for PrimeNG ofcourse.

**Comment by matheuspoleza** [comment]
@vazh totally agree with you. Maybe the problem was take so much care about ng2. A possible way as you mentioned is focus on ng1 and take the first release

**Comment by cyberwombat** [comment]
Along that thread here are a couple of repos for Semantic/Ng2
- https://github.com/lon-yang/angular2-semantic-ui
- https://github.com/edcarroll/ng2-semantic-ui

Not posting the ones with jQuery as that's not a full integration. 

I would love to see some energy towards a formal Ng2 support - a lot of Angular devs are just biting their nails waiting for the final release before pouncing on it. I personally would be super excited to use Semantic for it but am worried that these devs are going to lose steam without more support (already looks like #2 is inactive).

**Comment by vazh** [comment]
https://github.com/vladotesanovic/ngSemantic
probably the best one.

though i did say to start on ng1, but if its going to take a long time to decide, its better to jump ahead into angular2. its much easier anyway.

**Comment by cyberwombat** [comment]
@vazh unfortunately that one is a jQuery and not a native Angular one.

**Comment by vazh** [comment]
@cyberwombat yeah, but its still the best out there.

im curious as to how the heck PrimeNG do it.

**Comment by jmls** [comment]
so, there is _no_ further development in this project ? Does anyone know of any other project for angular 1 and semantic-ui?

**Comment by vazh** [comment]
@jmls if you actually read the comments you can actually find related projects to your question.

**Comment by douglasg14b** [comment]
@vazh Perhaps instead of being distasteful in the comments you could be helpful?

I don't see a single link to an active and updated Angular 1 project in this thread.
